djangocms_googlemap/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '2.1.1'
+__version__ = '2.2.0'
```

Comment:
I think maybe we should mention it at [CHANGELOG file](https://github.com/django-cms/djangocms-googlemap/blob/master/CHANGELOG.rst)?

Reply:
Makes sense :-)  ... and done